packages/react/src/components/UIShell/SideNavLinkText.tsx

```diff
@@ -20,16 +28,4 @@ function SideNavLinkText({ className: customClassName, children, ...rest }) {
   );
 }
 
-SideNavLinkText.propTypes = {
-  /**
-   * Provide the content for the link text
-   */
-  children: PropTypes.node.isRequired,
-
-  /**
-   * Provide an optional class to be applied to the containing node
-   */
-  className: PropTypes.string,
-};
-
```

Comment:
Can you just put it back the propTypes please. We still need that even with typescript. 
Other than that LGTM.

Reply:
Will do :) Can you elaborate on why tho? (out of curiosity)

Reply:
Done: 017ebd6

Reply:
It's useful for folks using JavaScript.
Here it is a few docs from React and from Carbon for more context:
https://react.dev/reference/react/Component#static-proptypes
https://github.com/carbon-design-system/carbon/blob/main/docs/guides/adding-component-types.md#strategy